db/migrate/20210406090131_add_answers_to_question.rb

```diff
@@ -0,0 +1,5 @@
+class AddAnswersToQuestion < ActiveRecord::Migration[6.1]
+  def change
+    add_belongs_to :answers, :questions, foreign_keys: true 
```

Comment:
Since this migration is present only on your branch (this whole feature being under development, it's safe to assume nobody else was curious to try it locally) you could change the `ref_name` from `questions` to `question` and remove the `20210406092054_fix_answer_question_fk` migration.

To do this, you should first rollback all migrations until the one previous to this. You could run multiple `rails db:rollback` commands, or a single one with an additional argument - `rails db:rollback STEP=6` which will rollback 6 migrations.

We have to do this because migration runs are stored in the `migrations` table in the database. Removing a file will desynchronise that table with the structure of `db/migrate`. Also, if we're changing existing migrations, we should make sure our changes work as intended!

An easier solution, especially since we're at the start, would be to drop the database completely, recreate it and then run the new set of migrations over it. We'd do this by running the `rails db:drop` command followed by `rails db:create` and lastly `rails db:migrate`